### PR TITLE
Add core weekly 2 of 2021

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 4th to Sunday 10th of January 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+PrestaShop [1.7.7.1](https://build.prestashop.com/news/prestashop-1-7-7-1-maintenance-release/) has been released! don't forget to update as this patch version fixes 41 regressions reported on version 1.7.7.0.
```
Comment (Progi1984) [inline news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]:
```suggestion
PrestaShop [1.7.7.1](https://build.prestashop.com/news/prestashop-1-7-7-1-maintenance-release/) has been released! Don't forget to update as this patch version fixes 41 regressions reported on version 1.7.7.0.
```

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next delivery of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2.
```
Comment (Progi1984) [inline news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]:
```suggestion
The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next delivery of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2 in maximum six weeks.
```

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+### Traces
+* [#15](https://github.com/PrestaShop/traces/pull/15): Remove old author statement, by [@matks](https://github.com/matks)
+* [#14](https://github.com/PrestaShop/traces/pull/14): Associated contributions to categories for each contributor, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Catalog statistics module
+* [#18](https://github.com/PrestaShop/statscatalog/pull/18): Remove olds files. Thank you [@okom3pom](https://github.com/okom3pom)
+
+
+### The PrestaShop open source project
```
Comment (Progi1984) [inline news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]:
```suggestion
### PrestaShop open source project
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md (hunk 2 above)
Comment (matks) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: I'm not sure about this. We should check with 1.7.7.2 product manager if 1.7.7.2 is already approved and scheduled?

Also 6 weeks is a maximum duration 😉  we can deliver sooner if needed

Comment (ttoine) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: then, "...in maximum six weeks."

**Review by ttoine — APPROVED** [review]

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md (hunk 2 above)
Comment (Progi1984) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: Yes, it is already approved. For schedule, 6 weeks since last release (so yesterday)

Comment (PierreRambaud) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: Already approved, I'm still working on an issue for the 1.7.7.2 and we have to fix autoupgrade problems :)

Comment (matks) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: OK. I then create the tracker issue https://github.com/PrestaShop/PrestaShop/issues/22783

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+PrestaShop [1.7.7.1](https://build.prestashop.com/news/prestashop-1-7-7-1-maintenance-release/) has been released! Don't forget to update as this patch version fixes 41 regressions reported on version 1.7.7.0.
+
+The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next delivery of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2.
```
Comment (matks) [inline news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]:
```suggestion
The kanban for `1.7.7` branch is not empty yet, it still contains unresolved regressions reported on version 1.7.7.0. The next batch of bugfixes for 1.7.7 will be delivered in patch version 1.7.7.2, which is scheduled to be delivered within [the next 6 weeks](https://devdocs.prestashop.com/1.7/project/release/minor-release-lifecycle/).
```

File: news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md (hunk 2 above)
Comment (Progi1984) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: I change my suggestion above.

Comment (matks) [reply news/_posts/core-weekly/2021-01-12-coreweekly-02-2021.md]: Already did

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review

**Pull request merged**